### PR TITLE
Added basemax_finance, xena_finance, mummy_finance, voodoo_trade, bmx and meridian to perpetual_trades

### PR DESCRIPTION
**Pull request by ARDev097**
Added trades data for basemax_finance on base.

### CODE REVIEW

**Comment by Hosuke** [comment]
Hi @ARDev097,
in future please try to keep `1 PR <-> 1 project` at a time.
Thank you.

**Comment by ARDev097** [comment]
Hello @Hosuke ,
I'll make sure to keep one PR per project moving forward.

File: models/voodoo_trade/base/voodoo_trade_base_schema.yml [hunk 1]
```diff
@@ -0,0 +1,89 @@
+version: 1
```
Comment (Hosuke) [inline models/voodoo_trade/base/voodoo_trade_base_schema.yml:1]: I think we may keep all .yml version as `2`:
https://docs.getdbt.com/reference/project-configs/version#yml-property-file-versions

**Review by Hosuke — CHANGES_REQUESTED** [review]

File: models/voodoo_trade/base/voodoo_trade_base_schema.yml (hunk 1 above)
Comment (ARDev097) [reply models/voodoo_trade/base/voodoo_trade_base_schema.yml]: Hello @Hosuke,
I have updated version `1` to version `2` in all .yml files. Please review and let me know if anything else is required from my end.

**Review by Hosuke — APPROVED** [review]
LGTM.✅
Thank you @ARDev097 

[Sample basemax_finance_base_perpetual_trades](https://dune.com/queries/3395340)
[Sample mummy_finance_base_perpetual_trades](https://dune.com/queries/3395343)
[Sample bmx_base_perpetual_trades ](https://dune.com/queries/3395344)
[Sample xena_finance_base_perpetual_trades](https://dune.com/queries/3395346)
[Sample voodoo_trade_base_perpetual_trades](https://dune.com/queries/3395349)
[Sample meridian_base_perpetual_trades](https://dune.com/queries/3395353)

**Review by jeff-dude — APPROVED** [review]

**Pull request merged**